en/get-started.md

```diff
@@ -860,4 +860,14 @@ Instructions for destroying the Kubernetes clusters depend on where the Kubernet
     kubectl delete namespace tidb-cluster
     ```
 
+5. Stop `kubectl` port forwarding
+
+    If you still have running `kubectl` processes that are forwarding ports, end them.
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    pgrep -lfa kubectl
```

Comment:
Do we need to mention how users can end the processes? The command here just lists the processes.